shared/src/main/scala/Elements.scala

```diff
@@ -189,15 +189,21 @@ object Elements:
       "a: lst -> int(a, 2), using list of digits"
     )(NumberHelpers.fromBinary)
 
-    addFull(
+    val tobinary = addVect(
       Monad,
       "b",
       "Convert To Binary",
       List("to-binary", "dec->bin", "decimal->bin"),
-      true,
       "a: num -> convert a to binary",
       "a: str -> bin(ord(x) for x in a)"
-    )(NumberHelpers.toBinary)
+    ) {
+      case a: VNum => NumberHelpers.toBinary(a)
+      case a: String =>
+        VList(
+          a.map(x => StringHelpers.chrord(x.toString))
+            .map(NumberHelpers.toBinary(_))*
```

Comment:
```suggestion
          a.map(x => NumberHelpers.toBinary(StringHelpers.chrord(x.toString)))*
```

Reply:
I could have sworn I tried something like that last night and couldn't get it to compile. Does this compile and mean that I'm just bad at scala? :p